tests/test_flow/generate_test_compatibility_cache.py

```diff
@@ -0,0 +1,68 @@
+# This script generates cache for a flow represented in Harness class
+# inside test_dir (tests/test_flow/test_persistence_compatibility).
+# The generated cache is used by test_persistence_compatibility.py tests
+# to validate that the cache can be deserialized by current Bionic.
+# In case the caching has changed, this file is used to replace the
+# test cache.
+#
+# To renegerate cache, run the following command from bionic/ dir
+#   `python -m tests.test_flow.generate_test_compatibility_cache`
+#
+# Note that the repo ignores *.pkl datafiles which is bypassed using
+# "Test data" section in .gitignore.
+
+import os
+
+from ..helpers import count_calls
+
+import bionic as bn
+
+
+# the folder structure should be supported by pytest-datadir
```

Comment:
This comment is out of date now, right?

Reply:
ah yes, removing the comment but I'm going to keep this folder structure